server/app/assets/javascripts/enumerator.ts

```diff
@@ -169,21 +171,18 @@ function setFocusAfterEnumeratorRemoval() {
  * inputs).
  */
 function refreshAddButtonStatus() {
-  const enumeratorQuestion = assertNotNull(
-    document.querySelector('.cf-question-enumerator'),
+  const enumeratorInputValues = Array.from(
+    document.querySelectorAll(
+      '.cf-question-enumerator input[data-entity-input]',
+    ),
+  ).map((item) => (item as HTMLInputElement).value)
+
+  // validate that there are no empty inputs.
+  const addButton = <HTMLInputElement>(
```

Comment:
suggestion (non-blocking):  Is there a need to cast this button as an input element?  Also, It would be best to stay consistent in the way you cast.  So, either using `as` or `<>`, but not mixing both.